.env.staging

```diff
@@ -40,9 +40,10 @@ VITE_ARROW_MODEL_URL="https://migration.geoportail.lu/static-ngeo/models/arrow5.
 VITE_ELEVATION_URL="https://migration.geoportail.lu/raster"
 
 # Auth
-VITE_LOGIN_URL="https://migration.geoportail.lu/login"
-VITE_LOGOUT_URL="https://migration.geoportail.lu/logout"
-VITE_USERINFO_URL="https://migration.geoportail.lu/getuserinfo"
+VITE_CREDENTIALS_ORIGIN="same-origin"
+VITE_LOGIN_URL="/login"
+VITE_LOGOUT_URL="/logout"
+VITE_USERINFO_URL="/getuserinfo"
```

Comment:
It won't work anyway because migration does not accept cross origin. Beside it looks like .env.staging is not used anyway ( did not find time to search why)